CHANGELOG.md

```diff
@@ -139,18 +141,6 @@ This release requires the 0.7.4.0 release of the PureScript compiler. Previous v
 - More instances for `ExceptT`
 - `ErrorT` has been removed in favor of `ExceptT`.
 
-## [v0.7.0-rc.2](https://github.com/purescript/purescript-transformers/releases/tag/v0.7.0-rc.2) - 2015-08-25
```

Comment:
I've removed the release candidates and merged their contents into the proper releases as we don't usually include them in changelogs.